CONTRIBUTING.md

```diff
@@ -131,8 +131,10 @@ Some common make targets are:
   & everything builds in the correct manner.
 - `make check-stage1-std NO_REBUILD=1` - test the standard library without
   rebuilding the entire compiler
-- `make check TESTNAME=<path-to-test-file>.rs` - Run a single test file
-- `make check-stage1-rpass TESTNAME=<path-to-test-file>.rs` - Run a single
+- `make check TESTNAME=<name-of-test-to-run>` - Run a single test file
+  - `TESTNAME` should be the fully qualified name of the test function, e.g.
```

Comment:
Yes this is specifically [implemented here](https://github.com/rust-lang/rust/blob/28bed3f5e64dfc083dc193412b65d95533a61d72/src/libtest/lib.rs#L993-L1001)